### ISSUE TITLE
Feature Request in Log Levels

### ISSUE THREAD

**Issue by kanayak123** [cannatag/ldap3#259]
PROTOCOL, NETWORK, EXTENDED log levels capture all the values, including encoded password which can be easily decoded. This creates a security concern of knowing other users password, for those who have access to the logs. This creates a hesitation in enabling this log level in a production environment, even for the debugging. 

Whereas, ERROR, BASIC log levels doesn't capture any user information which makes it difficult to related which log was for which account.

Its a feature request.

Either removed the encoded password value capture when `ldap3.utils.log.set_library_log_hide_sensitive_data` is set to `True` even when the log level is set to PROTOCOL, NETWORK, EXTENDED.
Or include basic details such as user DN or CN in the ERROR, BASIC log levels.

**Comment by cannatag** [comment]
do you means: >>     modification=PartialAttribute:

>>      type=b'unicodePwd'
>>      vals=Vals:
>>       0x220052006300350036003700390065006600670068002200

**Comment by kanayak123** [comment]
Yes. Example, my code updates mostly only unicodePwd attribute in the AD. So when I set the log level to PROTOCOL, NETWORK, EXTENDED it captures the utf-16-le encoded password in the logs. I agree its a value and might need for debugging. But, its a sensitive data. I suggest this value should be hidden when `set_library_log_hide_sensitive_data` is set to `True`. 


`DEBUG:ldap3:PROTOCOL:MODIFY request <{'entry': 'CN=Search User,OU=users,OU=IT,DC=example,DC=local', 'changes': [{'operation': 2, 'attribute': {'value': ['"\x00J\x00H\x00g\x00s\x00j\x00h\x00g\x006\x007\x005\x007\x006\x005\x003\x005\x006\x004\x00%\x00$\x00%\x00"\x00'], 'type': 'unicodePwd'}}]}> sent via <`

**Comment by cannatag** [comment]
Will work on this in version 2.2.
